docs/concepts/internationalization.md

```diff
@@ -51,15 +51,15 @@ Localization methods
 
 - `get(String, Object...)` localizes in the client's locale with formatting. The first parameter is a translation key, and the rest are formatting arguments for `String.format(String, Object...)`.
 
-### `TranslatableComponent`
+### `TranslatableContents`
 
-`TranslatableComponent` is a `Component` that is localized and formatted lazily. It is very useful when sending messages to players because it will be automatically localized in their own locale.
+`TranslatableContents` is a `ComponentContents` that is localized and formatted lazily. A `Component` can be created using `Component#translatable`. It is very useful when sending messages to players because it will be automatically localized in their own locale.
```

Comment:
Should this mention `MutableComponent.create(ComponentContents)` as well, for manually creating a `Component` with an instance of `TranslatableContents`?

Reply:
Perhaps not since the factory already exists. Maybe mention it for custom component purposes? Also, A MutableComponent is returned from Component#translatable; that should be specified instead of just Component.